equinix/resource_fabric_cloud_router.go

```diff
@@ -15,47 +18,250 @@ import (
 
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-func resourceCloudRouter() *schema.Resource {
+func FabricCloudRouterPackageSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"code": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Fabric Cloud Router package code",
+		},
+	}
+}
+func FabricCloudRouterAccountSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_number": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Optional:    true,
+			Description: "Account Number",
+		},
+	}
+}
+func FabricCloudRouterProjectSch() map[string]*schema.Schema {
```

Comment:
Lowercase method names because we don't need to export them. As shown in first comment.